atd-etl/cris_import/cris_import.py

```diff
@@ -83,21 +87,25 @@ def main():
     GRAPHQL_ENDPOINT_KEY = secrets["graphql_endpoint_key"]
     SFTP_ENDPOINT_SSH_PRIVATE_KEY = secrets["sftp_endpoint_private_key"]
 
+    S3_EXTRACT_BUCKET = secrets["s3_extract_bucket"]
+
     local_mode = False
     if bool(glob.glob("/app/development_extracts/*.zip")):
         local_mode = True
 
     zip_location = None
+    database_location = None
     if not local_mode:  # Production
-        zip_location = download_archives()
+        zip_location, database_location = download_s3_archive()
```

Comment:
Drop in replacement function that handles getting the archives. There's an extra bit of information that comes out of this step now which is where the SQLite DB file is saved.